### DISCUSSION TITLE
Screenshot path outside of project's directory

### DISCUSSION THREAD

**Discussion by gmetais** [macbre/phantomas#909] [Q&A] [answered]
Hi @macbre,

Currently, it is not possible to save a screenshot outside of the project's directory:

https://github.com/macbre/phantomas/blob/5fb3f06cace67b40cd03caaa790cc6d935b4e609/extensions/screenshot/screenshot.js#L38

I'm planning on running Phantomas on AWS Lambda. Virtual machines there have no hard drive and the only writable folder is /tmp.

Is there any reason why we have this limitation about the screenshot path option? Security mabye? Can we get rid of it?

I suggest to change the way it works. Phantomas could allows relative paths just like now, but also detect absolute paths that start with "/".

Should I make a pull request?

**Comment by macbre** [comment, marked as answer]
Great idea. Please prepare a pull request. 👌🏻
